data/meson.build

```diff
@@ -1,36 +1,40 @@
+# Install icons
 icon_sizes = ['16', '24', '32', '48', '64', '128']
 
 foreach i : icon_sizes
     install_data (
-      join_paths('icons', i + '.svg'),
-      install_dir: join_paths(get_option('datadir'), 'icons', 'hicolor', i + 'x' + i, 'apps'),
-      rename : meson.project_name() + '.svg'
+        join_paths ('icons', i + '.svg'),
+        install_dir: join_paths (get_option ('datadir'), 'icons', 'hicolor', i + 'x' + i, 'apps'),
+        rename : meson.project_name () + '.svg'
+    )
+    install_data (
+        join_paths ('icons', i + '.svg'),
+        install_dir: join_paths (get_option ('datadir'), 'icons', 'hicolor', i + 'x' + i + '@2', 'apps'),
+        rename : meson.project_name () + '.svg'
     )
 endforeach
 
-i18n.merge_file(
-    input: meson.project_name() + '.desktop.in',
-    output: meson.project_name() + '.desktop',
-    po_dir: join_paths(meson.source_root(), 'po', 'extra'),
+# Translate and install our .desktop file so the Applications Menu will see it
+i18n.merge_file (
+    input: meson.project_name () + '.desktop.in',
+    output: meson.project_name () + '.desktop',
+    po_dir: join_paths (meson.source_root (), 'po', 'extra'),
     type: 'desktop',
     install: true,
-    install_dir: join_paths(get_option('datadir'), 'applications')
+    install_dir: join_paths (get_option ('datadir'), 'applications')
 )
 
-i18n.merge_file(
-    input: meson.project_name() + '.appdata.xml.in',
-    output: meson.project_name() + '.appdata.xml',
-    po_dir: join_paths(meson.source_root(), 'po', 'extra'),
+# Translate and install our .appdata.xml file so AppCenter will see it
+i18n.merge_file (
+    input: meson.project_name () + '.appdata.xml.in',
+    output: meson.project_name () + '.appdata.xml',
+    po_dir: join_paths (meson.source_root (), 'po', 'extra'),
     install: true,
-    install_dir: join_paths(get_option('datadir'), 'metainfo')
-)
-
-install_data(
-    meson.project_name() + '.gschema.xml',
-    install_dir: join_paths(get_option('datadir'), 'glib-2.0', 'schemas')
+    install_dir: join_paths (get_option ('datadir'), 'metainfo')
 )
 
-install_data(
-    meson.project_name() + '.contract',
-    install_dir: join_paths(get_option('datadir'), 'contractor')
```

Comment:
Why remove the contract install logic but keep the contract file? 
I know the contract file will no longer work: https://github.com/elementary/contractor/issues/34. Maybe we should just remove all the contract logic in a separate PR?

Reply:
You're right. I'll add it back so it can be removed in another PR